best-practices/setting_up_rails_projects.md

```diff
@@ -14,9 +14,8 @@ Use the [config](https://github.com/railsconfig/config) gem for configuration ma
 ## Continuous Integration
 Projects should have implemented continuous integration. The following services are used often:
 
- - [travis-ci](https://travis-ci.org/) for running test suites
- - [hound-ci](https://houndci.com/repos) for style checking
- - [codeclimate](https://codeclimate.com/) for code coverage
+ - [travis-ci](https://travis-ci.com/) or [CircleCI](https://circleci.com/) for running test suites
```

Comment:
should we put forth a preference, maybe with brief explanation?  seems like infra team, at least, is moving more towards circleci lately, since it's a little more amenable to spinning up a bunch of supporting services via docker, and also a little more amenable to building docker images as needed?

Reply:
@jmartin-sul I don't have strong leanings here. @sul-dlss/infrastructure-team ? @sul-dlss/access-team ?

Reply:
i also don't have strong leanings (and if there is no actual preference, i'd just stick with the proposed language as-is)